wagtail/admin/templates/wagtailadmin/admin_base.html

```diff
@@ -54,8 +52,8 @@
     <script src="{% versioned_static 'wagtailadmin/js/core.js' %}"></script>
     <script src="{% versioned_static 'wagtailadmin/js/vendor.js' %}"></script>
     <script src="{% versioned_static 'wagtailadmin/js/wagtailadmin.js' %}"></script>
-
-    {% main_nav_js %}
+    <script src="{% versioned_static 'wagtailadmin/js/telepath/telepath.js' %}"></script>
```

Comment:
Should we bump telepath above wagtailadmin.js - it seems to be a pretty core import

Reply:
I tried to do this but got errors in the console – decided to keep the import order as-is for the time being. Ideally I think there should be fewer of those files to start with?